src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/enable-serverless-monitoring-aws-lambda-legacy.mdx

```diff
@@ -392,10 +392,12 @@ module.exports.handler = newrelic.setLambdaHandler((event, context, callback) =>
 9. Optional: To enable logging in serverless mode, set these environment variables:
     * Set `NEW_RELIC_LOG_ENABLED` to `true`.
     * Set `NEW_RELIC_LOG` to `stdout` for output to CloudWatch, or set to any writeable file location.
-    * The log level is set to `info` by default. See [other log levels](/docs/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration#logging_config).
+    * `NEW_RELIC_LOG_LEVEL` is set to `info` by default, and it's only used when sending function logs in your Lamba. See [other log levels](/docs/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration#logging_config).    
 10. Invoke the Lambda at least once. This creates a CloudWatch log group, which must be present for the next step to work.
 
 Our wrapper gathers data about the Lambda execution, generates a JSON message, and logs it to CloudWatch Logs. Next you'll [configure CloudWatch to send those logs to New Relic](/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/instrument-your-own/#manual-stream-logs).
+
+Read more on how to [send function logs with Lambda](https://github.com/newrelic/newrelic-lambda-extension/blob/c5c96bd8310156ce209ab55d5bc91fd084b3a8dd/config/config.go#L43).
```

Comment:
We should probably have a docs link here rather than linking to code, though I'm not sure if a doc exists that details all of the Extension's environment variables.

Reply:
@keegoid-nr - in docs we link to the GH repo. I've updated the link anyways.